src/components/survey/survey.ts

```diff
@@ -9,7 +10,17 @@ export interface SurveyOptions extends BaseOptions<string> {
 }
 
 export class SurveyComponent extends BaseComponent<SurveyOptions> {}
-export class SurveyElement extends BaseElement<SurveyComponent> {}
+export class SurveyElement extends BaseElement<SurveyComponent> implements OnInit{
+    public valueObj: Object;
+    public getValue(label: string, value: string): Object {
+        this.valueObj[label] = value;
```

Comment:
This makes no sense that you would be setting a value within a ```getValue``` method. This seems sloppy to me.